examples/local-minio/config_postgres.yaml

```diff
@@ -0,0 +1,48 @@
+repo:
+  repoUrl: s3+http://127.0.0.1:9000/datasets
+  caching:
+    registryCacheEnabled: true
+    metadataLocalFsCachePath: /tmp/kamu-metadata-cache
+uploadRepo:
+  maxFileSizeMb: 50
+  storage:
+    kind: s3
+    bucketS3Url: s3+http://127.0.0.1:9000/upload
+database:
+  provider: postgres
+  host: localhost
+  port: 5433
+  credentialsPolicy:
+    source:
+      kind: rawPassword
+      userName: root
+      rawPassword: root
+  databaseName: kamu
+auth:
+  didEncryption:
+    enabled: true
+    encryptionKey: hUuFbsNDOr9Gq6iRpCpilCueBaBWwbzM
+  providers:
+    - kind: github
+      clientId: 361a3b4fda86d0234d2f
+      clientSecret: 465849325236ed49253993744069e1bec6808554
```

Comment:
Maybe make a note that these are demo secrets, not real secrets. It catches the eye.